vmdb/app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -52,4 +52,10 @@ def textual_ems
     end
     h
   end
+
+  def textual_containers
+    num_of_containers = @record.number_of(:containers)
+    label = "Containers"
+    {:label => label, :image => "container", :value => num_of_containers}
```

Comment:
@abonas we usually link these back to a screen to show list of items, i.e you can link this to list of containers in Containers explorer